lib/usb/usb_f103.c

```diff
@@ -81,6 +81,7 @@ static void stm32f103_set_address(usbd_device *dev, uint8_t addr)
 
 /**
  * Set the receive buffer size for a given USB endpoint.
+ * Valid for F102, F103 and F0 devices
```

Comment:
no need for this here, it's already out of date, and is actually "good for the synopsis dwc usb core used here" or whatever the name is. just leave this out.